### PR TITLE
Added feature to allow downloading chrome driver on run time!

### PR DESCRIPTION
**Pull request by praveendvd**
Updated Read me to include details:

Now you can download chromedriver from your scripts like 

```
var chromedriver = require('chromedriver');
process.env.detect_chromedriver_version = true;
process.env.chromedriver_force_download = true;
process.env.chromedriver_download_to_target_folder = "C:/Users/output";



chromedriver.download().then(()=>{
console.log("completed")})
```

### CODE REVIEW

File: install.js [hunk 1]
```diff
@@ -337,3 +337,9 @@ function Deferred() {
   }.bind(this));
   Object.freeze(this);
 }
+
+
+if (require.main === module) {
```
Comment (giggio) [inline install.js:342]: `require.main` could be unavailable if using esm. See https://nodejs.org/api/esm.html#esm_differences_between_es_modules_and_commonjs and https://stackoverflow.com/a/60309682/2723305.

File: lib/chromedriver.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
+const { rejects } = require('assert');
```
Comment (giggio) [inline lib/chromedriver.js:1]: why is `rejects` here?

File: lib/chromedriver.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
 const fs = require('fs');
+const { resolve } = require('path');
```
Comment (giggio) [inline lib/chromedriver.js:3]: why is `resolve` here?

File: lib/chromedriver.js [hunk 4]
```diff
@@ -45,3 +47,9 @@ exports.stop = function () {
     exports.defaultInstance.kill();
   }
 };
+
+exports.download = async function () {
+  
+  return require("./../install.js")()
```
Comment (giggio) [inline lib/chromedriver.js:53]: can simplify it to `return require("../install.js")()`. Also, missing `;`.

File: install.js [hunk 5]
```diff
@@ -337,3 +337,9 @@ function Deferred() {
...
+  install()
+} 
+module.exports = install
```
Comment (giggio) [inline install.js:345]: export the function by name. `module.exports = { install }`. And missing `;`.

**Review by giggio — CHANGES_REQUESTED** [review]

**Comment by praveendvd** [comment]
Hi Please see the updated changes, thank you for the detailed review  :)

**Pull request closed without merging**